specs/consensus.md

```diff
@@ -32,7 +32,7 @@ Consensus Rules
 | `VERSION`                             | `uint64` | `1`     |         | Version of the LazyLedger chain. Breaking changes (hard forks) must update this parameter.                                                                  |
 | `CHAIN_ID`                            | `uint64` | `1`     |         | Chain ID. Each chain assigns itself a (unique) ID.                                                                                                          |
 | `NAMESPACE_ID_BYTES`                  | `uint64` | `32`    | `byte`  | Size of namespace ID, in bytes.                                                                                                                             |
-| `NAMESPACE_ID_RESERVED_BYTES`         | `uint64` | `2`     | `byte`  | Size of reserved namespace ID range, in bytes.                                                                                                              |
+| `NAMESPACE_ID_MAX_RESERVED`           | `uint64` | `65535` |         | Value of maximum reserved namespace ID (inclusive). 2 bytes worth of IDs.                                                                                   |
```

Comment:
Wouldn't one byte be sufficiently large for reserved namespaces?

Reply:
Maybe? It doesn't really affect anything either way if we use hashes to assign namespace IDs. Just through it would be better to go liberal so we don't have to change it later, but the argument could be made to not be needlessly wasteful.

Reply:
If you feel anything stronger than pure apathy at the choice, then I can change it to 255 😂

Reply:
Reduced in 7d08046.

Reply:
I didn't feel strong about it. I was just wondering.